ethers-providers/src/transports/quorum.rs

```diff
@@ -685,6 +682,95 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_get_quorum_block_number() {
+        let mut providers = Vec::new();
+        let mut mocked = Vec::new();
+
+        for value in [100, 101, 68, 100, 102] {
+            let mock = MockProvider::new();
+            for _ in 0..6 {
```

Comment:
this 6 because there are 6 expected RPCs to each provider (i.e. 6 calls to `get_quorum_block_number` in this test) right?

Reply:
Yes, you push each time you want it to have a response, otherwise it gives a no response error as part of the mock implementation. (I do use this fact in one of the tests).